src/core_modules/capture-core/components/WidgetRelatedStages/WidgetRelatedStages.container.tsx

```diff
@@ -51,7 +51,15 @@ export const WidgetRelatedStagesPlain = ({
         linkedEvent,
         isLoading: isLinkedEventLoading,
     } = useLinkedEventByOriginId({ originEventId: eventId, skipBidirectionalChecks: true });
-    const relatedStageRef = useRef<?RelatedStageRefPayload>(null);
+    const relatedStageRef = useRef<{
+        eventHasLinkableStageRelationship:() => boolean;
+        formIsValidOnSave: () => boolean;
+        getLinkedStageValues: () => {
+            linkMode?: any;
+            relatedStageDataValues: any;
+            selectedRelationshipType: any;
+        };
+            }>(null);
```

Comment:
import and use `RelatedStageRefPayload`. like this:

`import { type RelatedStageRefPayload } from './index';`
` const relatedStageRef = useRef<RelatedStageRefPayload>(null);`